### PR TITLE
Endpoint investigation plan enhancement

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-5957)

## Description
Updated the playbook to perform investigation only on the alerts associated with the incident.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml [hunk 1]
```diff
@@ -2217,7 +2215,7 @@ inputs:
           conditionInBetween: {}
           else:
             value:
-              simple: ${alert= '(initiatorsha256:"' + val.initiatorsha256 + '" or agentid:"' + val.agentid + '") ' + 'and sourceBrand:"' + val.sourceBrand + '" and name:"' + val.name + '"'}
+              simple: ${alert= '(initiatorsha256:"' + val.initiatorsha256 + '" or hostip:"' + val.hostip + '") ' + 'and sourceBrand:"' + val.sourceBrand + '" and name:"' + val.name + '"'}
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml:2218]: change back

File: Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml [hunk 2]
```diff
@@ -2230,13 +2228,13 @@ inputs:
...
             value:
-              simple: ${alert= '(filesha256:"' + val.filesha256 + '" or agentid:"' + val.agentid + '") ' + 'and sourceBrand:"' + val.sourceBrand + '" and name:"' + val.name + '"'}
+              simple: ${alert= '(filesha256:"' + val.filesha256 + '" or hostip:"' + val.hostip + '") ' + 'and sourceBrand:"' + val.sourceBrand + '" and name:"' + val.name + '"'}
   required: false
   description: The query for searching previous alerts based on the file we want to respond to. Decided by the If-Then-Else expression wether it's the initiator or the target file.
   playbookInputQuery:
 - key: ShouldOpenTicket
   value:
-    simple: "False"
+    simple: "True"
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml:2237]: change back

File: Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml (hunk 1 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml]: Done

File: Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml (hunk 2 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml]: Done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**